src/main/java/catan/services/impl/PlayServiceImpl.java

```diff
@@ -125,12 +138,12 @@ private void endTurn(GameBean game) throws GameException {
             Integer newPreparationCycle = game.getPreparationCycle();
             boolean preparationFinished = newPreparationCycle == null && !game.getCurrentMove().equals(1);
             shouldUpdateNextMove = (preparationFinished || previousPreparationCycle.equals(newPreparationCycle));
+        } else {
+            game.setDiceThrown(false);
```

Comment:
I have changed it in the same way as in early 67 story, now it was changed. while merging the implementation from 67 would be inserted